integration_tests/__init__.py

```diff
@@ -82,6 +73,13 @@ def run_snapcraft(self, command, project_dir=None, yaml_dir=None,
         if not os.getenv('SNAPCRAFT_IGNORE_APT_AUTOREMOVE', False):
             self.addCleanup(self.run_apt_autoremove)
 
+        if os.path.exists('parts'):
+            self.parts_dir = 'parts'
+        if os.path.exists('stage'):
+            self.stage_dir = 'stage'
+        if os.path.exists('prime'):
+            self.prime_dir = 'prime'
```

Comment:
I wanted them in __init__, so we are on the same page. doing `copy_project_to_cwd` instead of defining `project_dir` in every test is already a win.